### DISCUSSION TITLE
Better documentation for community testing

### DISCUSSION THREAD

**Discussion by CaseyZS** [ni/actor-framework#63] [General]
**Is your feature request related to a problem? Please describe.**
Taking a look at this for the first time, it's unclear to me what to do in order to test the latest code/builds.

**Describe the solution you'd like**
A simple text document would go a long way as far as describing what someone who wants to help needs to do (e.g. installation instructions, etc.) and then maybe some scenarios that could serve as good initial tests.

**Describe alternatives you've considered**
There the AF Guild Discord, but finding the relevant information in there is difficult compared to having clear, concise instruction within the repo itself.

**Additional context**
N/A

**Comment by svelderrainruiz** [comment]
I created a method to crowdsource community testing [here](https://github.com/ni/actor-framework/discussions/100)
